repos/system_upgrade/common/models/grubconfigerror.py

```diff
@@ -3,7 +3,15 @@
 
 
 class GrubConfigError(Model):
+    ERROR_CORRUPTED_GRUBENV = 'corrupted grubenv'
+    ERROR_MISSING_NEWLINE = 'missing newline'
+    ERROR_GRUB_CMDLINE_LINUX_SYNTAX = 'GRUB_CMDLINE_LINUX syntax'
```

Comment:
that's nice actually. However I am not sure whether this is allowed tbh. It's the first time (IIRC) we set any attribute for a model classes and we strongly required to have present only fields + topic in the class. But I really like it. so I want to be sure, we all agree to have attributes allowed in models, so we do not enforce people to use a different design in such cases (@oamg/developers ping).